packages/node_modules/@node-red/editor-client/src/sass/palette-editor.scss

```diff
@@ -305,3 +305,11 @@ button.red-ui-palette-editor-upload-button {
         margin-left: 10px;
     }
 }
+
+button.red-ui-update-status {
+    width: auto;
+    padding: 0 3px;
+    span {
+        color: var(--red-ui-text-color-warning);
+    }
```

Comment:
```suggestion
```

Propose not using colour for this - I don't think we use colour elsewhere in the core for this sort of thing.

Reply:
I don't know if without color the widget will be visible to users. What worries me is that they won't notice it.

`nrlint` use this color but not the core.

Reply:
We don't need to draw lots of attention - just having the widget there if there are updates available is a step forward from what we have today.